manager/assets/modext/widgets/element/modx.grid.template.tv.js

```diff
@@ -129,14 +133,43 @@ Ext.extend(MODx.grid.TemplateTV,MODx.grid.Grid,{
         }
         return m;
     }
+
     ,updateTV: function(itm,e) {
-        MODx.loadPage('Element/TemplateVar/Update', 'id='+this.menu.record.id);
+        MODx.loadPage('element/tv/update', 'id='+this.menu.record.id);
```

Comment:
Why did you change this? 
To prevent issues it's better to have MODX connector actions match the classname of the corresponding processor (CamelCase).

This also applies to the other action urls in this PR.

Reply:
The `Element/TemplateVar/Update` does not work, `element/tv/update` is the same action in the TV tree and used it.
As for the register, I did not know which one to use, because both uses in MODX 3, I created an issue https://github.com/modxcms/revolution/issues/14744 on this topic.

Reply:
Most, in my opinion, use lowercase actions, so I used lowercase.

Reply:
Actions should be in camel case, matching the name/structure of the processor class. The lowercase actions work because MODX (tries to) converts them camelcase. 

For reference also see this PR: https://github.com/modxcms/revolution/pull/14871

If only the lowercase actions works we need to figure out why the camel case version doesn't work and fix it.

Reply:
Note that this loads a **controller**, not a **processor**. Controllers haven't been touched yet for 3.0.

Reply:
Actions work in any case. In the examples, they just are not correctly indicated:
- `Element/**TemplateVar**/Update` -> `Element/**TV**/Update` or `element/tv/update`
-` security/access/usergroup/**namespace**/remove` -> `Security/Access/UserGroup/**AccessNamespace**/Remove` or `security/access/usergroup/accessnamespace/remove`

Reply:
And the bugs associated with https://github.com/modxcms/revolution/issues/14639 are due to the fact that Linux considers files in a different register - different (files exactly).
As I understand it, but maybe I'm wrong.
But now actions work both in lower and upper case (the server is the same).

Reply:
Just to reiterate, #13639 is about **processors**. This fix is about a **controller**. Totally different thing. 

Uppercasing paths is new to **processors** only. Processors also still work with lowercase paths. Processors also convert legacy "TV" to new "TemplateVar".

**Controller** behavior has not changed in 3.0 (yet). [That splits the path and uppercases the first character regardless of how it is provided.](https://github.com/modxcms/revolution/blob/3.x/core/src/Revolution/modManagerResponse.php#L264-L274) That could use CamelCase if we'd wanted, but it doesn't have to. 

(Your fix in this line of code, specifically converting `TemplateVar` back to `tv` is correct)

Reply:
> Note that this loads a **controller**, not a **processor**. Controllers haven't been touched yet for 3.0.

Aha! Conversation solved. 😄